src/_locales/sv/messages.json

```
@@ -54,5 +54,38 @@
   },
   "btnFullScreen": {
     "message": "Visa i helskärm"
+  },
+  "kbdShortcuts": {
+    "message": "Tangentbordsgenvägar"
+  },
+  "kbdMovement": {
+    "message": "Rörelsenycklar"
+  },
+  "kbdFeature": {
+    "message": "Navigationsnycklar för geografiska egenskaper"
```

```suggestion
    "message": "Navigationsnycklar för geografiska objekt"
```

Similar to https://github.com/Maps4HTML/mapml-extension/pull/43#pullrequestreview-1072549067.